CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ BUG FIXES:
 
 * Releases targeting Windows now have a `.exe` suffix (#1291).
 * Adaptively recover from dirty and corrupted git repositories in cache (#1279).
+* Fix `-no-vendor` flag for `ensure -update` ([#1361](https://github.com/golang/dep/pull/1361))
```

Comment:
A period (`.`) at the end and the full link is not required, just #1361 is enough.